pkg/cli/start.go

```diff
@@ -1242,6 +1247,17 @@ func maybeWarnMemorySizes(ctx context.Context) {
 	}
 }
 
+func cleanupOldTempDirs(specs []base.StoreSpec) error {
```

Comment:
Could we extract out the code that we already have in `initTempStorageConfig` and use that in `cleanupOldTempDirs`? The existing code looks like it handles encrypted stores too.

That way in `runStartInternal` we're not doing a second round of unnecessary cleanup.